catalog/core/catalog-core-directorymonitor/src/main/java/org/codice/ddf/catalog/content/monitor/ContentDirectoryMonitor.java

```diff
@@ -74,32 +105,55 @@ public ContentDirectoryMonitor(CamelContext camelContext) {
      * also called whenever an existing route is updated.
      */
     public void init() {
-        LOGGER.trace("INSIDE: init()");
-
         if (routeCollection != null) {
             try {
                 // This stops the route before trying to remove it
                 LOGGER.debug("Removing {} routes", routeCollection.size());
                 camelContext.removeRouteDefinitions(routeCollection);
             } catch (Exception e) {
-                LOGGER.info("Unable to remove Camel routes from Content Directory Monitor", e);
+                LOGGER.warn("Unable to remove Camel routes from Content Directory Monitor", e);
             }
         } else {
             LOGGER.debug("No routes to remove before configuring a new route");
         }
 
-        org.codice.ddf.security.common.Security.runAsAdmin(() -> {
-            configureCamelRoute();
+        runAsAdmin(this::configure);
+    }
+
+    private Object configure() {
```

Comment:
Is this simply so that it becomes a callable? In mine when i have a callable I find it nice to return a bool instead, true for success, false for failure. 

Although if you don't want that and still prefer using the null returns you should at least make your return type `Void` instead of `Object` as it is more technically accurate.